R/Sound_DMR.R

```diff
@@ -566,21 +566,21 @@ run_binomial <- function(LM, i = int, formula,
 #' @inheritParams run_binomial
 #' @param individual_name_z (str) - *optional* the name of the z column when running
 #' individual DMR analysis
-#' @return Output_Frame (df) - updated with the summary data from the model
+#' @return Input_frame (df) - updated with the summary data from the model
 #' @export
 #'
 
-run_model <- function(data, i, Output_Frame, formula, model_type,
+run_model <- function(data, i, Input_frame, formula, model_type,
```

Comment:
The doc string for this method still looks off to me. For that matter the `run_binomial` doc string looks off too.

Reply:
ok, looking at this again, i might be wrong and r might be doing something smarter than i was originally thinking

Reply:
Yeah, I just checked. As much as I know, that looks fine and we don't have to match the parameter names exactly. When the function is called, if they line up its all good.